### ISSUE TITLE
Should "master and slave" terminology be replaced in the documentation?

### ISSUE THREAD

**Issue by Felix-Dev** [MicrosoftDocs/winrt-api#1678]
As the title states.

Some links of potential interest to check:
* [Wikipedia](https://en.wikipedia.org/wiki/Master/slave_(technology)#Terminology_concerns)
* [Python Language](https://www.vice.com/en_us/article/8x7akv/masterslave-terminology-was-removed-from-python-programming-language)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d728a96f-a076-e884-f1fb-1d432da2cad9
* Version Independent ID: e15a1901-835d-9342-1ca8-198255c6957a
* Content: [SpiDevice.TransferFullDuplex(Byte\[\], Byte\[\]) Method (Windows.Devices.Spi) - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/windows.devices.spi.spidevice.transferfullduplex?view=winrt-19041#Windows_Devices_Spi_SpiDevice_TransferFullDuplex_System_Byte___System_Byte___)
* Content Source: [winrt-api-build/xml/Windows.Devices.Spi/SpiDevice.xml](https://github.com/MicrosoftDocs/winrt-api/blob/docs/winrt-api-build/xml/Windows.Devices.Spi/SpiDevice.xml)
* Product: **uwp**
* Technology: **winrt-reference**
* GitHub Login: @stevewhims
* Microsoft Alias: **stwhi**

**Comment by stevewhims** [comment]
Hi, @Felix-Dev, and thanks for filing this issue. Yes, I'd be happy to make corrections here. I've started some investigation to reach out to the feature team to see what terminology we can use. I suspect it'll be along the lines of devices that control and devices that are controlled, but I'll make sure we use terms whose meanings are clear. (Meanwhile, btw, I saw a PR that does amend this topic, so I'll take that change into account, too).

**UPDATE**: I have changes in the publishing pipeline. Ordinarily, those would be live within 24 hours. But it looks like we're having auth issues with the publishing pipeline so I'll keep my eye on these changes and if they're not live early next week I'll escalate. Thanks for your patience.

Thanks!
Steve
